code/modules/mob/living/living_defense.dm

```diff
@@ -58,6 +58,10 @@
 
 /mob/living/bullet_act(obj/projectile/P, def_zone, piercing_hit = FALSE)
 	. = ..()
+	//SKYRAT EDIT START
```

Comment:
as according to the modularization guidelines, please be adding some sort of phrase or key to let people know why this was edited: like you did with `//SKYRAT EDIT: BULLET_ACT_FORCE_PIERCE_BLOCK`

Reply:
That's a good idea, thanks!